spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialConvolutionMap.scala

```diff
@@ -37,8 +38,9 @@ class SpatialConvolutionMap[@specialized(Float, Double) T: ClassTag](
   val dW: Int = 1, // The step of the convolution in the width dimension.
   val dH: Int = 1, // The step of the convolution in the height dimension
   val padW: Int = 0, // The additional zeros added per width to the input planes.
-  val padH: Int = 0 // The additional zeros added per height to the input planes.
-
+  val padH: Int = 0, // The additional zeros added per height to the input planes.
+  val wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out?